doc/source/_toc.yml

```
@@ -59,29 +59,35 @@ parts:
       - file: train/train
         title: Ray Train
         sections:
-          - file: train/getting-started
-            title: "Getting Started"
           - file: train/key-concepts
             title: "Key Concepts"
-          - file: train/user-guides
-            title: "User Guides"
+          - file: train/distributed-pytorch
+            sections:
+              - file: train/distributed-pytorch/converting-existing-training-loop
+              - file: train/distributed-pytorch/data-loading-preprocessing
+              - file: train/distributed-pytorch/using-gpus
+              - file: train/distributed-pytorch/monitoring-logging
+              - file: train/distributed-pytorch/persistent-storage
+                title: Configuring Persistent Storage
```

It came up in conversation today that it might feel more natural to have the order of these two switched:
```suggestion
              - file: train/distributed-pytorch/persistent-storage
                title: Configuring Persistent Storage
              - file: train/distributed-pytorch/monitoring-logging
```

As a user I might think about "configurations" together (scale, storage), and then "reporting" together (metrics, checkpoints)